<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAdmin</class>
 <widget class="QDialog" name="WindowAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="Img.qrc">
    <normaloff>:/new/prefix36/img/task.png</normaloff>:/new/prefix36/img/task.png</iconset>
  </property>
  <widget class="QLabel" name="Fon">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>961</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgb(54, 57, 67);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="leftside" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>61</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="upside" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_PageName">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>158</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixel Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Адміністратор</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <family>Fixel Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_email">
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnclose">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(://img/white_x_circle_icon.png);
	background-color: transparent;
	background: none;
	border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="searchEmployee">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>8</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid rgb(30, 101, 172);
padding-left: 20px;


}</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="baseside" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>211</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>207</width>
          <height>407</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_tasks" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>491</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{


}</string>
   </property>
   <widget class="QScrollArea" name="scrollArea_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scroll_widget_main">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>489</width>
       <height>409</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgb(30, 101, 172);
border-radius: 20px;
color: rgb(255, 255, 255);
font-size: 15px; 
font-weight: bold;
}
QPushButton:hover{
background: rgb(255, 255, 255);
color: rgb(37, 125, 214);
}</string>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgb(30, 101, 172);
border-radius: 20px;
color: rgb(255, 255, 255);
font-size: 15px; 
font-weight: bold;
}
QPushButton:hover{
background: rgb(255, 255, 255);
color: rgb(37, 125, 214);
}</string>
   </property>
   <property name="text">
    <string>Змінити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgb(30, 101, 172);
border-radius: 20px;
color: rgb(255, 255, 255);
font-size: 15px; 
font-weight: bold;
}
QPushButton:hover{
background: rgb(255, 255, 255);
color: rgb(37, 125, 214);
}</string>
   </property>
   <property name="text">
    <string>Видалити</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
